packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -341,7 +341,7 @@ class WritingFlow extends Component {
 		const focusableNodes = focus.focusable.find( this.container );
 		const target = findLast( focusableNodes, isTabbableTextField );
 		if ( target ) {
-			placeCaretAtHorizontalEdge( target, true );
+			placeCaretAtHorizontalEdge( target, true, { preventScroll: true } );
```

Comment:
What does this fix?

Reply:
It is a workaround the default behaviour in Chrome which brings the focused element into view, which in our case scrolls up [the whole document](https://github.com/WordPress/gutenberg/pull/15165#issuecomment-498849250)

Reply:
That seems... fine? Isn't the caret supposed to scroll into view? How is this related to the space at the bottom?

Reply:
Yes, except that it scrolls wrong in Chrome when clicking the bottom editor space, if we apply these changes: 

https://user-images.githubusercontent.com/1202812/58569372-fb0da900-8203-11e9-8be2-866ca3c9f0f5.gif

The caret is not into view because Chrome scrolls too much :)